starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -18,7 +18,7 @@ plugins {
 @if (features.language().isJava()) {
     id "com.diffplug.eclipse.apt" version "3.22.0"
 }
-@if (features.language().isKotlin() || features.testFramework().isKotlinTest()) {
+@if (features.language().isKotlin() || features.testFramework().isKotlinTest() || features.testFramework().isKoTest()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411